znai-reactjs/src/doc-elements/docMeta.js

```diff
@@ -37,4 +41,11 @@ function getDocId() {
     return docMeta.id
 }
 
-export {setDocMeta, isPreviewEnabled, getDocId}
+function getSupportLink() {
+    const support = docMeta.support
+    return support && support.link ?
+        new Promise((resolve, reject) => resolve(support.link))
+        : jsonPromise("/support/" + docMeta.id).then(supportMeta => supportMeta.link)
```

Comment:
What will happen for the docs outside, that don't define `support` inside `docMeta.json`? Will there still be an attempt to make `GET` call?

Reply:
Yes, it will try to get it from the backend service.

Reply:
Every docs hosted outside like on GitHub pages will have an error in the console and network tab. I don't have a good answer to this yet, but it is something I would prefer to avoid.

Reply:
Can you make a change, so I can customize the area like footer.md?

Reply:
It is a good point. In theory it is already possible, but is not convenient to do so. 
Remember `themeRegistry.override` method? That is how you can or replace components to render parts of docs. I don’t think `Footer` is exposed through the `defaultElementsLibrary`. 
I will need to think about it.